backend/services/videoPromptService.js

```diff
@@ -0,0 +1,174 @@
+const { VideoPromptCreateDto } = require("../dtos/videoPromptCreateDto");
+const PromptSpace = require("../models/promptSpaceModel");
+const { PROMPT_SPACE_TYPE } = require("../types/promptSpaceTypes");
+const AppError = require("../utils/appError");
+const VideoPrompt = require("../models/videoPromptModel");
+const {
+    getPresignedUrlForGet,
+    createObjectKeyFromVideo,
+    uploadFileToS3,
+} = require("./awsS3Service");
+const { PaginationResponseDto } = require("../dtos/paginationResponseDto");
+const {
+    GetAllVideoPromptsInputDto,
+    VideoPromptDto,
+} = require("../dtos/videoPromptDto");
+
+const generateVideoPromptFullMessage = (input) => {
+    const { message, samplingSteps, cfgScale, eta, fps } = input;
+    return `${message}. Settings: samplingSteps=${samplingSteps}, cfgScale=${cfgScale}, eta=${eta}, fps=${fps}`;
```

Comment:
Don't need the others included here as below it was already used.

Reply:
I am supposed to delete it, do I?

Reply:
Yea, I created the video version generateVideoPromptFullMessage, to include descripption as full message send to the AI generation api, and if the api supports by default then i didn't include. Like width and height.